packages/thriver-accounts/lib/templates/profile.html

```diff
@@ -7,7 +7,7 @@
 		{{/each}}
         <input type="submit" value="{{submitValue}}">-->
 
-        {{> quickForm collection="Meteor.users" id="updateProfile" doc=doc type="update" 
-            omitFields=omitFields fields="profile" }}
+        {{> quickForm collection="Meteor.users" id="updateProfile" doc=doc type="update"
+            omitFields=omitFields fields="profile" template="series" }}
```

Comment:
You should either refrain from committing this part until templating is completed, or move these changes into a separate feature branch.